usaspending_api/references/abbreviations.py

```diff
@@ -0,0 +1,63 @@
+code_to_state = {
```

Comment:
Hmm... this is a tough call!

I am usually all "third-party modules FTW", and this looks like a cool one (I'll keep it in mind for the future).  On the other hand, it makes our dependencies a little more complex (esp. since it has some dependencies of its own), and I don't think the state abbreviations are going to require ongoing maintenance.

I think I'll throw this out for discussion in #datastore!